src/components/trendLegend/trendLegend.js

```diff
@@ -0,0 +1,64 @@
+import React, {PureComponent} from 'react';
+import styled from 'styled-components';
+
+const TrendLegendDiv = styled.div`
+  width: 100%;
+  text-align:center;
+`
+
+class TrendLegend extends PureComponent {
+  render() {
+    const trend = this.getTrend();
+    const volume = this.getVolume();
+    const date = this.getPreviousDate();
+    const iconClass = `metric-trend fa fa-arrow-${trend}`
```

Comment:
Hmmm.. adding a font awesome dependency. ideally we need to pull out the styling from Dashboard to here, but ok for now, maybe leave a `@todo`